README.md

```diff
@@ -1,2 +1,39 @@
-# Personal-portfolio
-A list of the projects I have done over time
+# Personal-portfolio-project
+This project lists the projects I have done over time. It includes the source code for the website containing my portfolio and other supporting files. It was built for mobile platforms therefore you will get the best experience using the website. 
+
+##Built With   
+HTML and CSS
+
+## Getting Started
+This project can be acessed at my [Github account](https://github.com/Ghostman747) by anyone as it is a public repository.
+You will need a git hub account if you want to clone it and also a web browser capable of running HTML 5 for maximum quality of the content.
+The projects themselves can be accessed at the index.html file available in this repository.
+Also included is additional files for cheking the quality of the webpage and their installation directions.
+License information is also attached in this repository. Anyone is allowed to use the files accordingly as long as they use them positively.
+
+For any queries or suggestions, you can reach me at any of the channels listed below.
+
```

Comment:
- [ ] OPTIONAL: I recommend updating this section to make everything very clear, people are lazy creatures, they don't like to read 😆 . Let's make it as instructions, you can use this template: 

```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/Ghostman747/Personal-portfolio/.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/Ghostman747/Personal-portfolio/`.
- go to the repository folder in your command prompt and open `index.html` file

```

> You can customize it and add more steps 😉

Reply:
This one is perfect.